tests/shared/drive-constraints.spec.js

```diff
@@ -490,29 +657,53 @@ describe('Shared: DriveConstraints', function() {
 
       it('should return false if the drive is not large enough and is a source drive', function() {
         m.chai.expect(constraints.isDriveValid(this.drive, {
-          path: path.join(this.mountpoint, 'rpi.img'),
-          size: 5000000000
+          path: path.join(this.mountpoint, path.join(__dirname, 'rpi.img')),
```

Comment:
Are you sure it's fixed? Or is GH displaying an out-of-date diff for some reason? :-/